mobile/lib/main.dart

```diff
@@ -257,7 +257,7 @@ class _TenTenOneAppState extends State<TenTenOneApp> {
 
       subscribeToNotifiers(context);
 
-      runBackend(config).then((_) {
+      await runBackend(config).then((_) {
```

Comment:
🙃 if we `await` here we can move the `Flog.info` simply below the `runBackend(config);` line.

Reply:
I did it for you @Restioson